src/plugins/intel_cpu/src/nodes/kernels/registers_pool.hpp

```diff
@@ -0,0 +1,184 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
Ok, I will update the code according to the new code style. Yes, this code we decided to do as a separate PR, because it is a reusable part and we want our team members could use it asap. I did this pool as part of Gather JIT kernel refactoring and I am planning to add PR for this, so it will contain usage of this code.

Reply:
Is there any document describing the openvino naming code style? I couldn't find any,

Reply:
We don't have requirements for private plugin APIs.

We use `ncc` tool to check naming style for OpenVINO public API, but this tool is disabled for internal code, so at the current moment different plugins are using different naming styles inside.